packages/core/helper-plugin/lib/src/components/GenericInput/index.js

```diff
@@ -226,7 +226,7 @@ const GenericInput = ({
           hint={hint}
           name={name}
           onChange={(date) => {
-            const formattedDate = date.toISOString();
+            const formattedDate = date?.toISOString();
```

Comment:
Maybe worth adding some tests around `DateTimePicker` with the associated file? It's looking rather light at the moment.